src/lib/components/LatexBlock.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+	import { afterUpdate, onMount } from "svelte";
+
+	export let code = "";
+
+	onMount(() => {
+		let script = document.createElement('script');
+        script.src = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-svg.js";
+        document.head.append(script);  
+        script.onload = () => {
+            MathJax = {
+                tex: {inlineMath: [['$', '$'], ['\\(', '\\)']]},
+                svg: {fontCache: 'global'}
+            };
+		};					  
+	});
+</script>
+
+<div class="group relative my-4 rounded-lg">
+	<!-- eslint-disable svelte/no-at-html-tags -->
```

Comment:
Why do you disable eslint for this line if you're not using `@html` afterwards ?